src/anyvar/extras/vcf.py

```diff
@@ -130,12 +130,12 @@ def register_existing_annotations(
                     continue
                 if state == ".":
                     state = ""
-                seq_ref = SequenceReference(refgetAccession=refget_accession)
+                seq_ref = SequenceReference(refgetAccession=refget_accession)  # type: ignore (values that aren't specified default to None)
```

Comment:
I think it's fine to leave an explanation here but this problem is so ubiquitous throughout any usage of the GKS python libraries that we shouldn't plan on reciting this mantra every time we need to use it

It might also be helpful to just ignore the specific error code instead of a blanket ignore (but that'll also be annoying to do every time, idk)